adal/tests/Test.ADAL.NET.UIAutomation/Test.ADAL.NET.UIAutomation.csproj

```diff
@@ -32,10 +33,10 @@
     <PackageReference Include="Xamarin.UITest" Version="2.2.4" />
     <PackageReference Include="NUnitTestAdapter" Version="2.1.1" />
     <PackageReference Include="Newtonsoft.Json" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="2.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
```

Comment:
Why  downgrade these?

Reply:
@henrik-me newer versions ran into compatibility issues that prevented the build from passing on VSTS

Reply:
compatibility or missing components on the box?

---
In reply to: [199915776](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1109#discussion_r199915776) [](ancestors = 199915776)